Code/Engine/Foundation/Profiling/Profiling_Tracy.h

```diff
@@ -21,11 +22,36 @@ EZ_ALWAYS_INLINE ezUInt32 ___tracyGetStringLength(const ezStringBuilder& szStrin
 {
   return szString.GetElementCount();
 }
-
+EZ_ALWAYS_INLINE ezUInt32 ___tracyGetStringLength(const ezHashedString& szString)
+{
+  return szString.GetView().GetElementCount();
+}
+EZ_ALWAYS_INLINE const char* __convertezStringToConstChar(const ezString& szString)
+{
+  return szString.GetData();
+}
+EZ_ALWAYS_INLINE const char* __convertezStringToConstChar(const ezStringBuilder& szString)
+{
+  return szString.GetData();
+}
+EZ_ALWAYS_INLINE const char* __convertezStringToConstChar(const ezHashedString& szString)
+{
+  return szString.GetData();
+}
+EZ_ALWAYS_INLINE const char* __convertezStringToConstChar(const ezStringView& szString)
+{
+  ezStringBuilder temp_str;
+  szString.GetData(temp_str);
+  return temp_str.GetData();
```

Comment:
This is a typical use after free, you can't return the pointer to the stringbuilder data, when that stringbuilder is local to the function.

Is there no way to give tracy a string view?
Alternatively you would need something else to copy the strings to for a longer time